### DISCUSSION TITLE
建议增加几个强势的预设的优化，比如欢愉（打宇宙6）虚无（打宇宙3）和记忆（打宇宙7）

### DISCUSSION THREAD

**Discussion by XIKONGSAKET** [CHNZYX/Auto_Simulated_Universe#151] [Ideas]
练度不是很够的月卡党，勉勉强强打满混沌回忆，box非常全，每一个新角色都有，遗器很垃圾（这个真的没办法），AI的巡猎打哪个都不能稳定吃下首领，每次都在首领翻车全灭，看着非常糟心。相反，抽了卡夫卡的直接带一队辅助都能打穿宇宙3-5
既然用这个说明受众的练度都是不太高的，需要资源。如果本身队伍非常强了那么对这个脚本的需求就会下降

**Comment by CHNZYX** [comment]
想要怎样的优化呢？目前已经对每个命途的祝福优先级有了优化。
另外，丰饶宇宙5虽然慢，但是不吃练度

**Reply by XIKONGSAKET** [reply]
笑死，感谢作者大大回复。选虚无的时候，三个奇物：降维骰子、諨灵胶、毁灭火漆，居然选了毁灭火漆，hhhh，这就不知道是为什么了，明明你的优先选择里面已经把諨灵胶放在最前面了。原来低练度挂机选择是宇宙5嘛哈哈哈，这我倒是真不知道，不过建议作者把这点写在read me里面。
已经看出来了确实有在优化啦！

**Comment by XIKONGSAKET** [comment]
还有，能不能把遭遇设置为最低优先级，本来就不打箱子，碎片不够，也不怎么强化祝福，还不放秘技，很多遭遇都难度比较大的，打boss我的主C都没倒，遭遇把我卡夫卡干掉了，好几次看着它明明有战斗和事件可以选非得选遭遇。我可以在哪里改这个优先级？

**Comment by XIKONGSAKET** [comment]
dalao，能不能告诉我怎么把遭遇设置为最低优先级，本来就不打箱子，碎片不够，也不怎么强化祝福，还不放秘技，很多遭遇都难度比较大的，打boss我的主C都没倒，遭遇把我卡夫卡干掉了，好几次看着它明明有战斗和事件可以选非得选遭遇。我可以在哪里改这个优先级？

ZYX ***@***.***> 于2023年8月17日周四 18:56写道：
